tilequeue/query/__init__.py

```diff
@@ -72,12 +72,16 @@ def _make_rawr_fetcher(cfg, layer_data, query_cfg, io_pool):
         source_value = data['value']
         table_sources[tbl] = Source(source_name, source_value)
 
-    # set this flag, and provide a postgresql subkey, to generate RAWR tiles
-    # directly, rather than trying to load them from S3. this can be useful
-    # for standalone use and testing.
-    is_s3_storage = not rawr_source_yaml.get('generate-from-scratch')
-
-    if is_s3_storage:
+    # set type to "generate", and provide a postgresql subkey, to generate
+    # RAWR tiles directly, rather than trying to load them from S3. this can
+    # be useful for standalone use and testing.
+    #
+    # other types are:
+    #   s3 - to fetch RAWR tiles from S3
+    #   source - to fetch RAWR tiles from any tilequeue tile source
```

Comment:
should this comment read `generate-from-scratch` and `store`?

Reply:
Thanks for spotting that! Yes, fixed in 1dd0c1ce40dcbdad076f2aec7f017cd23bd7779d.